### DISCUSSION TITLE
Colorscheme of front page screenshot

### DISCUSSION THREAD

**Discussion by schickele** [akiyosi/goneovim#503] [General]
Sorry to bother you with this side question. Could you perhaps tell me the name of the vim colorscheme you used in the first screenshot of the Goneovim GitHub front page? The closest I could find was this one:

[https://vimcolorschemes.com/patstockwell/vim-monokai-tasty](https://vimcolorschemes.com/patstockwell/vim-monokai-tasty)

I’ve tested Goneovim earlier this year. I will stick to my old Gvim setting for now, but great work nonetheless, thanks a lot!

Greetings from Europe

Paul

**Comment by akiyosi** [comment]
Hi :D 
Sorry for the delay in answering.
Yes, It is vim-monokai-tasty.
